packages/grafana-data/src/events/EventBusContext.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { EventBusWithSource } from './EventBus';
+
+export const EventBusWithSourceContext = React.createContext<EventBusWithSource | undefined>(undefined);
```

Comment:
I think we should probably change this to either PanelContext or VisualizationContext?   (and make the context an object), so we can add more things than just the eventBus to it, for example: sharedTooltipMode, and other context flags/options.  

thoughts @dprokop @ryantxu  ?

Reply:
Pushed a commit using PanelContext so we can get an idea of how that would look.

Reply:
👍